runconfig/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	OnBuild         []string              // ONBUILD metadata that were defined on the image Dockerfile
 	Labels          map[string]string     // List of labels set to this container
 	StopSignal      string                `json:",omitempty"` // Signal to stop a container
+	Tmpfs           map[string]string     `json:",omitempty"` // List of tmpfs (mounts) used for the container
```

Comment:
I wonder if this should be in `HostConfig`. It looks like a very host specific flag that can break portability.

@cpuguy83 what do you think?